r/man/load_experiment_data.Rd

```diff
@@ -1,5 +1,5 @@
 % Generated by roxygen2: do not edit by hand
-% Please edit documentation in R/development_helpers.R
```

Comment:
what are these files?

Reply:
where do they get generated from?

Reply:
These are documentation files from roxygen, they are the documentation of our python package, explaining what each function does, etc.
Then when in Rstudio you do ?function you get help!